### PR TITLE
i18n(id): Update index.mdx translation

### PR DESCRIPTION
**Pull request by iyyafi**
#### Description
- Update ID translation for index.mdx file and add Testimonial section

### CODE REVIEW

File: docs/src/content/docs/id/index.mdx [hunk 1]
```diff
@@ -39,6 +46,114 @@ import AboutAstro from '~/components/about-astro.astro';
 	</Card>
 </CardGrid>
 
+<TestimonialGrid title="What people are saying">
```
Comment (HiDeoo) [inline docs/src/content/docs/id/index.mdx:49]: The "What people are saying" title can be translated.

**Review by HiDeoo — CHANGES_REQUESTED** [review]
Thanks for the new translation :star2:

As a non-native speaker, I reviewed this PR using the [Docs Translation Review Checklist](https://hackmd.io/1KYnrpUQR0SAGTzHIUoUDg?view). I left one comment regarding a missing translation but otherwise the translation looks good to me :thumbsup:

File: docs/src/content/docs/id/index.mdx (hunk 1 above)
Comment (iyyafi) [reply docs/src/content/docs/id/index.mdx:49]: Ah i miss that one. thank you for the review. will update soon.

**Review by HiDeoo — APPROVED** [review]
Thanks for translating the missing sentence, looks great to me now :rocket:

**Pull request merged**